forc/src/ops/forc_build.rs

```diff
@@ -170,7 +171,16 @@ pub fn build(command: BuildCommand) -> Result<pkg::Compiled> {
         let hash_file_name = format!("{}{}", &manifest.project.name, SWAY_BIN_HASH_SUFFIX);
         let hash_path = output_dir.join(hash_file_name);
         fs::write(hash_path, &bytecode_hash)?;
-        info!("  Bytecode hash is: {}", bytecode_hash);
+        info!("  Hash of the Script bytecode is: {}", bytecode_hash);
```

Comment:
Maybe just "Script bytecode hash: {}"?

This is the current format for contracts when running `forc deploy`:
```console
  Bytecode size is 112 bytes.
Contract id: 0x01f0c9a726086b74216f2c5ddaa13378c0636c8c926552bac8234b4a07c8344d
Logs:
TransactionId(HexFormatted(68a19c3ec223d8d276ad112892ec65780e9596b708921be98de78a8fd28486db))
```